docs/src/modules/reference/pages/migration-guide.adoc

```diff
@@ -1,3 +1,25 @@
 = Migration Guide
 
 include::ROOT:partial$include.adoc[]
+
+== SDK 3.3.0
+
+This release runs all user component and endpoint logic on Java 21 virtual threads, this means it is
+safe to do IO calls without blocking the underlying OS thread.
+
+This means it is no longer any need to use `CompletionStage` and `CompletableFuture` composition when
+doing component client, HTTP client and gRPC client calls. Where a service would previously call `.invokeAsync()` and get a future back, you can now instead use `.invoke()` and directly operate on or return the return value.
```

Comment:
The English here could be improved?

this means ... This means
return the return
do vs make call